[OSDOCS#17160] using-ruby CQA (#117479)
openshift/openshift-docs: applications/creating_applications/templates-using-ruby-on-rails.adoc

toc::[]

[role="_abstract"]

> You can build and deploy a Ruby on Rails 4 application on {product-title} by developing it locally, storing the source in Git, and deploying the database, frontend, and route services. This allows you to validate your application locally before deploying it to the cluster as a set of distinct services.

Reviewer: ```suggestion
You can build and deploy a Ruby on Rails 4 application on {product-title} by developing it locally.

Store the source in Git, then deploy the database, frontend, and route services. With this process, you can to validate your application locally before deploying it to the cluster as a set of distinct services.
```
shorter sentences for lower FK score; this abstract is much better, but i think break it up for the word count

[WARNING]
====

> You must complete each module in this assembly in order before you deploy your application on {product-title}. If a step fails, confirm that every preceding step completed successfully before you continue.

Reviewer: ```suggestion
You must complete each part of this tutorial in order to deploy your application on {product-title}. If a step fails, confirm that every preceding step completed successfully before you continue.
```
doc parts are invisible to users; always checkout AI output

====

== Prerequisites

> * You have basic Ruby and Rails knowledge.

Reviewer: ```suggestion
* You have basic Ruby on Rails knowledge.
```
Ruby on Rails is a separate thing from Ruby. I think this should probably be Ruby on Rails. Perhaps create a Jira to come back to this section?

> * You have Ruby 2.0.0+, Rubygems, and Bundler installed locally.

Reviewer: yeah, latest version is 4.0.6, so please make a Jira and mark on spreadsheet that this section needs update (or sunset as case may be)

Author: Link to Jira https://redhat.atlassian.net/browse/OSDOCS-21592

* You have basic Git knowledge.
ifndef::openshift-online[]
* You have a running instance of {product-title} 4.
endif::[]
ifdef::openshift-online[]

> * You have a provisioned account in OpenShift Online.

Reviewer: FYI OpenShift Online is obsolete. You can remove this content if you like. (But not required)

Author: I'm going to keep it here since it's in and ifdef/endif. don't want to delete and then have someone say what happened

endif::[]

> * You have the {oc-first} installed and can log in with your email address and password.

Reviewer: ```suggestion
* The {oc-first} installed.
* You are logged into a running {product-title} cluster.
```
